core/setup.py

```diff
@@ -49,7 +49,7 @@
     },
     install_requires=[
         "Jinja2==3.1.2",
-        "MarkupSafe>=0.23,<2.1",
+        "MarkupSafe>=2.0",
```

Comment:
Any sense in just removing it entirely? I don't think we had it here until we needed to add the upper boundary earlier this year (https://github.com/dbt-labs/dbt-core/pull/4746)

Reply:
Removing the pin entirely is the right move.  I wasn't aware of the history there.

It looks like dbt-labs/dbt-server([#48](https://github.com/dbt-labs/dbt-server/pull/48)) and dbt-labs/dbt-cloud ([#4928](https://github.com/dbt-labs/dbt-cloud/pull/4928)) both pinned it as well and can not remove their pins.